### DISCUSSION TITLE
The Oversight Committee is officially formed!

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/oc#6] [Announcements]
On the 19th of January 2023 the Steering Committee approved a motion to officially form the Oversight Committee with scope and  governance as defined in these docs:

- https://github.com/Green-Software-Foundation/oc/blob/main/oc-responsibilities.md
- https://github.com/Green-Software-Foundation/oc/blob/main/oc-responsibilities.md

The next steps are elections for the chair/vice-chair positions.

**Comment by tmcclell** [comment]
Congratulations!
